orttraining/orttraining/test/graph/bert_toy_fetches.h

```diff
@@ -4,13 +4,15 @@
 #pragma once
 
 #include <unordered_map>
+#include <array>
```

Comment:
Fix the warning:

orttraining\orttraining\test\graph\bert_toy_fetches.h(13,56): error 6262:  : Function uses '131072' bytes of stack:  exceeds /analyze:stacksize '16384'.. This allocation was for a compiler-generated temporary for 'double [16384]' at line 13.  Consider moving some data to heap.